openkat/settings.py

```diff
@@ -421,9 +422,12 @@ class SaferExceptionReporterFilter(SafeExceptionReporterFilter):
 
 # Turn on the browsable API by default if DEBUG is True, but disable by default in production
 BROWSABLE_API = env.bool("BROWSABLE_API", DEBUG)
+JWT_KEY = env.str("JWT_KEY", "")
```

Comment:
Why not reuse the already existing SECRET_KEY? We can copy the logic from drf-simplejwt because that also uses SECRET_KEY by default: https://django-rest-framework-simplejwt.readthedocs.io/en/latest/settings.html#signing-key